src/views/Rankings.vue

```diff
@@ -203,6 +206,11 @@ export default class RankingsView extends Vue {
   @Watch("searchModel")
   public onSearchModelChanged(rank: Ranking) {
     if (!rank) return;
+
+    if (!this.playerIsRanked(rank)) {
```

Comment:
this will break on the profile page, when the player is not real. Also we have players that are unranken (have 2/3 games) that could be found somehow differently.

Reply:
the player will only be returned if it has a rank and/or playeroverallstats, then it should get battletag and routing to profile page should work. do you have an example i can use to break the profile page?